stdlib/number.gr

```diff
@@ -404,6 +417,30 @@ provide let isInfinite = (x: Number) => {
   }
 }
 
+/**
+ * Determines weather two values are considered close to each other using a relative and absolute tolerance.
+ *
+ * @param a: The first value
+ * @param b: The second value
+ * @param relativeTolerance: The relative tolerance to use
+ * @param absoluteTolerance: The absolute tolerance to use
```

Comment:
These should probably be more descriptive because it's not clear what they do.

Reply:
I tried to improve these and got them a bit better but i dont know if they are good enough yet,

https://docs.python.org/3/library/math.html I was looking at python's docs for the description.